app/models/scanned_resource.rb

```diff
@@ -14,7 +14,10 @@ def to_solr(solr_doc = {})
       doc[ActiveFedora::SolrQueryBuilder.solr_name("ordered_by", :symbol)] += send(:ordered_by_ids)
       doc[ActiveFedora::SolrQueryBuilder.solr_name("logical_order", :symbol)] = [logical_order.order.to_json]
       doc[ActiveFedora::SolrQueryBuilder.solr_name("logical_order_headings", :stored_searchable)] = logical_order.object.each_section.map(&:label)
-      doc[ActiveFedora::SolrQueryBuilder.solr_name("collection", :facetable)] = in_collections.map(&:title)
+      in_collections.each do |col|
```

Comment:
We'll have to be careful with this - if things start to slow down a lot, this may be a culprit.